testing/e2e/skr-tester/pkg/command/update.go

```diff
@@ -41,7 +43,8 @@ func NewUpdateCommand() *cobra.Command {
 	skr-tester update -i instanceID -p planID --updateOIDC --customOIDC '{"clientID":"foo-bar","issuerURL":"https://new.custom.ias.com"}'  Update the instance with a custom OIDC configuration.
 	skr-tester update -i instanceID -p planID --updateMultipleOIDC  				   Update the instance with a predefined OIDC configurations list.
 	skr-tester update -i instanceID -p planID --updateAdministrators --customAdministrators admin1@acme.com,admin2@acme.com                Update the instance with custom administrators.
-	skr-tester update -i instanceID -p planID --updateAdditionalWorkerNodePools --customAdditionalWorkerNodePools '[{"name":"worker-1","machineType":"m6i.large","haZones":true,"autoScalerMin":3,"autoScalerMax":20}]'      Update the instance with custom additional worker node pools.`,
+	skr-tester update -i instanceID -p planID --updateAdditionalWorkerNodePools --customAdditionalWorkerNodePools '[{"name":"worker-1","machineType":"m6i.large","haZones":true,"autoScalerMin":3,"autoScalerMax":20}]'      Update the instance with custom additional worker node pools.
+	skr-tester update -i instanceID -p planID --ingressFiltering=true                      Update the instance with ingress filtering enabled.`,
```

Comment:
As in previous comment. We can update with true or false accordingly enabling and disabling ingress filtering.

Reply:
This is an example where `true` is set so i mentioned enablement.